### ISSUE TITLE
Can't connect bank if IOS screen reader is turned on

### ISSUE THREAD

**Issue by adnxy** [plaid/react-native-plaid-link-sdk#347]
# The problem

Our users reporting that they can't connect bank if the IOS screen reader is turned on.
They have to turn off the screen reader in order to connect to their bank.

# Environment

|                        |  |
| ------------------------| ---------- |
| Plaid Link React Native          | 5.0.2 |
| ReactNative Version     |  0.63.4 |
| Occurs on Android             | no   |
| Occurs on iOS     | yes  |
| iOS Version           | 11, 12, 13, 14 |

# Steps to Reproduce

When trying to connect bank (Michigan State FCU) it shows up in search, but after entering login info, it searches, tries to connect, then takes the user back to the home screen.
To make it work, the user has to turn off the screen reader on iPhone, and everything works well.

```
# Expected Result
Users should be able to connect their banks if the screen reader is turned on

# Screenshots
no screenshots

**Comment by afh** [comment]
Thank you for your report, @adnxy. We'll look into it.

**Comment by afh** [comment]
When using the latest version of Plaid Link React Native (7.1.1) which includes LinkKit 2.2.2 account linking suceeds when a screen reader (VoiceOver) is enabled.

@adnxy can you confirm?

**Comment by jhurray** [comment]
Bump @adnxy - I also confirmed this
